libqtile/command/base.py

```diff
@@ -170,6 +170,7 @@ def cmd_doc(self, name) -> str:
         """
         if name in self.commands:
             command = self.command(name)
+            assert command
```

Comment:
Ugly hack but it's here because `self.command` can return `None`.

Reply:
Hmm, I think mypy is right to complain here because we are running `self.command(name)` knowing that the return type is not going to be `None` (right?). In which case the assert makes sense to clarify this (to mypy as well as the reader). Alternatively it could `if None: continue` or something.

Reply:
It shouldn't be `None` as we've done `if name in self.commands:`. I think the `assert` is needed or we do a `# type: ignore`.

Reply:
Right - that's kinda what I was alluding to. *We* know that this cannot be `None` because we know we are passing in a `name` that will give us a non-`None` value. But the type system is saying "you're getting an `Optional[T]` here". Our assert is (rightfully) telling mypy "nah mate it's not a `None` don't worry", so I think it's fine with the assert.